airbyte-protocol/models/src/main/resources/protocol_models/airbyte_message.yaml

```diff
@@ -0,0 +1,78 @@
+---
+"$schema": http://json-schema.org/draft-07/schema#
+"$id": https://github.com/airbytehq/airbyte/blob/master/airbyte-protocol/models/src/main/resources/protocol_models/airbyte_message.yaml
+title: AirbyteMessage
+description:
+type: object
+additionalProperties: true
+required:
+- type
+properties:
+  type:
+    description: 'Message type'
+    type: string
+    enum:
+    - RECORD
+    - STATE
+    - LOG
+  record:
+    description: 'record message: the record'
+    "$ref": airbyte_message.json#/definitions/AirbyteSchema
+  state:
+    description: 'schema message: the state. Must be the last message produced. The platform uses this information'
+    "$ref": airbyte_schema.json#/definitions/AirbyteStateMessage
+  log:
+    description: 'log message: any kind of logging you want the platform to know about.'
+    "$ref": airbyte_message.json#/definitions/AirbyteLogMessage
+
+definitions:
+
+  AirbyteRecordMessage:
+    type: object
+    additionalProperties: false
+    required:
+    - data
+    - extracted_at
+    properties:
+      data:
+        description: 'the record data'
+        type: object
+        existingJavaType: com.fasterxml.jackson.databind.JsonNode
+      extracted_at:
```

Comment:
i think this might just be left over from singer. how about emitted_at? don't feel too strongly. just want to make sure we're picking this name because we like it best as opposed to because it was what singer did.